Assets/_Scripts/Managers/SoundManager.cs

```diff
@@ -1,31 +1,52 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.SceneManagement;
 
 public class SoundManager : MonoBehaviour
 {
-    public static SoundManager instance;
+    [SerializeField]
+    private AudioClip[] backgroundMusic;
+    private AudioClip sceneStartSound;
 
-    public AudioSource musicSource;
-    public AudioSource effectsSource;
-
-    public AudioClip backgroundMusic;
+    private AudioSource audioSource;
 
     private void Awake()
     {
-        
-    }
-    // Start is called before the first frame update
-    void Start()
-    {
-        
+        if (FindObjectsOfType<SoundManager>().Length > 1) //Destroys other Objects with the SoundManager Script if more than 1 is present
+        {
+            Destroy(gameObject);
+            return;
+        }
```

Comment:
This class should just be a `Singleton<SoundManager>` instead and be placed in the `Title` scene